dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -81,7 +81,12 @@ def manual_pass
     unless user
       flash[:alert] = "User (ID: #{params[:user_id]}) not found"
     end
-    script = Script.get_from_cache(params[:script_id_or_name])
+    script = nil
+    if params[:script_id_or_name].to_i.to_s == params[:script_id_or_name]
+      script = Script.find_by_id(params[:script_id_or_name])
+    else
+      script = Script.find_by_name(params[:script_id_or_name])
+    end
```

Comment:
Oh the question is "why", not "are we". For "why", because I tried fighting the tests and the tests won. And I don't see it mattering much.